jenkins-scripts/lib/_homebrew_github_commit.bash

```diff
@@ -8,54 +8,51 @@
 PR_URL_export_file=${PR_URL_export_file:-${WORKSPACE}/pull_request_created.properties}
```

Comment:
In first 1-5 lines of the file we tell what paremeter we do expect for this file to work, needs update.